lib/tasks/braintree_tokens.rake

```diff
@@ -0,0 +1,8 @@
+namespace :champaign do
+  desc "Create braintree payment method tokens out of existing token columns"
+  task braintree_tokens: :environment do
+    Payment::BraintreeCustomer.all.each do |customer|
+      Payment::BraintreePaymentMethod.create!({customer_id: customer.customer_id, token: customer.card_vault_token})
```

Comment:
this seems a little tricky because before you run the migrations, the `Payment::BraintreePaymentMethod` table doesn't exist, but after you run it, `customer.card_vault_token` doesn't exist. I think you may need to run `rake db:migrate:redo VERSION=my_version` with the version being the timestamp of the migration to make the PaymentMethod table